### PR TITLE
fn: enable failing test back

### PR DESCRIPTION
**Pull request by skinowski**

### CODE REVIEW

**Comment by rdallman** [comment]
🤞

**Comment by skinowski** [comment]
@rdallman  not ready. :-) I'm going to be doing a series of rebuilds, etc. to make sure...

**Comment by rdallman** [comment]
fixes #822

File: api/agent/drivers/docker/docker.go [hunk 1]
```diff
@@ -145,6 +145,8 @@ func (drv *DockerDriver) Prepare(ctx context.Context, task drivers.ContainerTask
 			Image:        task.Image(),
 			Volumes:      map[string]struct{}{},
 			OpenStdin:    true,
+			AttachStdout: true,
```
Comment (skinowski) [inline api/agent/drivers/docker/docker.go]: no change here, just being explicit.

**Comment by skinowski** [comment]
@rdallman this is now ready, rebuild circle ci without any issues a few times..

Also modified limited writer for stderr/stdout safety and added some tests for it...

**Pull request merged**